docs/usage.md

```diff
@@ -64,6 +64,18 @@ For single-sample peaking calling without a control sample, leave the second col
 ### `--broad`
 Run MACS with the `--broad` flag. With this flag on, MACS will try to composite broad regions in BED12 ( a gene-model-like format ) by putting nearby highly enriched regions into a broad region with loose cutoff. The broad region is controlled by the default qvalue cutoff 0.1.
 
+### `--saturation`
+Running saturation analysis by sub-sampling the sequence reads of ChIP sample from 10% to 100% with 10% interval, then calling peaks with MACS.
+
+### `--blacklist_filtering`
+Filtering ENCODE blacklisted regions from ChIP-seq peaks.
```

Comment:
Would be good if this is a little longer and more detailed, _eg._

> "Specifying this flag instructs the pipeline to use bundled ENCODE blacklist regions to filter out known blacklisted regions in the called ChIP-seq peaks. Please note that this is only supported when `--genome` is set to..." 

or whatever